ocaml/xapi/helpers.ml

```diff
@@ -2041,11 +2041,26 @@ let update_ca_bundle =
           )
     )
 
+(** {pp_hash hash} outputs the hexadecimal representation of the {hash}
+  adding a semicolon between every octet, in uppercase.
+*)
+let pp_hash hash =
```

Comment:
Please change, previously it was obvious that it was meant to pretty-print the certificate hash because of the module it was contained, but that's not the case anymore